src/recommendations/ProductCard/index.jsx

```diff
@@ -75,11 +74,12 @@ const ProductCard = ({
     ],
   );
   const handleCardClick = () => {
-    trackRecommendationCardClickOptimizely(userId?.toString());
     trackRecommendationsClicked(
-      product.courseKey,
-      false,
+      product.uuid,
+      product.cardType,
```

Comment:
does cardType always contain the correct type? is it confirmed?

Reply:
Yes, It will be always.